tee-worker/litentry/core/stf-task/receiver/src/lib.rs

```diff
@@ -179,11 +180,20 @@ where
 			.recv()
 			.map_err(|e| Error::OtherError(format!("receiver error:{:?}", e)))?;
 
+		let start_time = std::time::Instant::now();
+
 		match &req {
 			RequestType::IdentityVerification(req) =>
 				IdentityVerificationHandler { req: req.clone(), context: context.clone() }.start(),
 			RequestType::AssertionVerification(req) =>
 				AssertionHandler { req: req.clone(), context: context.clone() }.start(),
 		}
+
+		if let Err(e) = context.ocall_api.update_metric(EnclaveMetric::StfCallObserveExecutionTime(
+			Box::new(req),
+			start_time.elapsed().as_secs_f64(),
+		)) {
+			warn!("Failed to update metric for top pool size: {:?}", e);
```

Comment:
I think this message is not relevant. It's stf execution metric.

Reply:
Done